guides/source/active_job_basics.md

```diff
@@ -124,8 +124,68 @@ GuestsCleanupJob.perform_later(guest1, guest2, filter: 'some_filter')
 
 That's it!
 
+### Bulk Enqueuing
+
+You can Enqueue multiple jobs at once using [`perform_all_later`][]. Bulk Enqueuing reduces the number of round trips to a queue data store like Redis.
+
+`perform_all_later` is a top-level api on ActiveJob. It accepts instantiated jobs as arguments (note that this is different from `perform_later`). `perform_all_later` does call `perform` under the hood. The arguments passed to `new` will be passed on to `perform` when it's eventually called.
+
+Here is an example calling `perform_all_later` with `GuestCleanupJob` instances:
+
+```ruby
+# Create jobs to pass to `perform_all_later`. The arguments to `new` are
+# passed on to `perform`
+guest_cleanup_jobs = User.pluck(:id).map { |id| GuestsCleanupJob.new(guest_id: id) }
+
+# Will enqueue a seperate job for each instance of `GuestCleanupJob`
+ActiveJob.perform_all_later(guest_cleanup_jobs)
+```
+
+`perform_all_later` logs the number of jobs successfully enqueued, something like `Enqueued 3 jobs to Async (3 GuestsCleanupJob)` if it was called with `ActiveJob.perform_all_later(job1, job2, job3)`. 
+
+The return value of `perform_all_later` is `nil`. Note that this is different from `perform_later`. This may be enhanced to return the number of successfully enqueued job in the future, but for now it's `nil`.
```

Comment:
(suggestion, non-blocking): Not sure that the last part of the sentence is needed.

```suggestion
The return value of `perform_all_later` is `nil`. Note that this is different from `perform_later`. This may be enhanced to return the number of successfully enqueued job in the future.
```